src/main/java/app/view/FileMenuBar.java

```diff
@@ -30,8 +44,41 @@ public FileMenuBar(App app)
         });
         file.getItems().add(newFile);
 
+
+        /* Open file menu*/
+        /*
+        Menu openFile = new Menu("Open");
+        for(File f: filterErrorFiles(getResourceFolderFiles("")))
+        {
+            MenuItem fileOption = new MenuItem(f.getName());
+            openFile.getItems().add(fileOption);
+            fileOption.setOnAction(e -> app.gotoSimulation(f.getName()));
+        }
+        file.getItems().add(openFile);
+        */
+
+
+        FileChooser fileChooser = new FileChooser();
+        String directory = FilePath.get("");
+        fileChooser.setInitialDirectory(new File(directory));
         MenuItem openFile = new MenuItem("Open");
-        openFile.setOnAction(e -> System.out.println("Test open..."));
+
+        // create an Event Handler
+        EventHandler<ActionEvent> eventOpen =
+                new EventHandler<ActionEvent>() {
+
+                    public void handle(ActionEvent e)
+                    {
+                        // get the file selected
+                        File file = fileChooser.showOpenDialog(app.getStage());
+
+                        if (file != null) {
+                            app.gotoSimulation(file.getName());
+                        }
+                        System.out.println("clicked on openFile");
+                    }
+                };
```

Comment:
Although this is a perfectly valid way of achieving this, consider next time using the lambda form (which was there previously) and defining a functional method:
```java
openFile.setOnAction(e -> eventOpen(e));
```
 ...
```java
private void eventOpen(ActionEvent e)
{
    // do stuff ...
}
```

This keeps the code far cleaner, and allows other functions to be added using the same code (for example if we wanted a ctrl-o shortcut to open a file, it would call the same method)